### PR TITLE
small fix so the map will load properly when editing nodes

### LINKED ISSUE #87: Back end: Fix map display on venue tab

### PR DESCRIPTION
**Pull request by markconroy**
Closes #87

### CODE REVIEW

**Comment by finnlewis** [comment]
Tested locally and this works great for me @markconroy , thank you! 

Not sure why the tests are failing though.

**Comment by finnlewis** [comment]
I wonder if this is because there is not yet a 2.0.0 release of localgov? 

In which case we want to get this resolved https://github.com/localgovdrupal/localgov/pull/238

Which also depends on this: https://github.com/localgovdrupal/localgov/pull/209

@ekes @stephen-cox @andybroomfield @Adnan-cds any thoughts?

File: localgov_directories.module [hunk 1]
```diff
@@ -444,3 +444,12 @@ function localgov_directories_remove_block_from_content_type_sidebar(string $blo
 
   return TRUE;
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function localgov_directories_form_alter(&$form, &$form_state, $form_id) {
+  if ($form_id === 'node_localgov_directories_venue_form') {
```
Comment (Adnan-cds) [inline localgov_directories.module]: Should this go into the [localgov_directories_venue](https://github.com/localgovdrupal/localgov_directories/tree/2.x/modules/localgov_directories_venue) submodule?

Comment (markconroy) [reply localgov_directories.module]: Yes, that might make more sense.

**Comment by stephen-cox** [comment]
@markconroy Tests should be fixed with #102. Feel free to merge the `hotfix/fix-github-ci-for-2.0.0` branch into this to get the tests running properly.

**Comment by markconroy** [comment]
@Adnan-cds This is now part of localgov_directories_venue module.

File: localgov_directories.module (hunk 1 above)
Comment (Adnan-cds) [reply localgov_directories.module]: Thanks Mark.  Nearly there.  There are some coding standard issues flagged up by the Github Test runner.  If you can please have a [quick look](https://github.com/localgovdrupal/localgov_directories/pull/101/checks?check_run_id=2674980160#step:4:41).  Shouldn't take more than a few minutes to resolve.

**Comment by stephen-cox** [comment]
Just a couple of coding standards errors and then the tests will pass:

```
 FILE: ...algov_directories_venue/localgov_directories_venue.libraries.yml
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 6 | ERROR | [x] Expected 1 newline at end of file; 0 found
----------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------


FILE: ...les/localgov_directories_venue/localgov_directories_venue.module
----------------------------------------------------------------------
FOUND 1 ERROR AND 1 WARNING AFFECTING 2 LINES
----------------------------------------------------------------------
 4 | ERROR   | [x] Doc comment short description must be on the first
   |         |     line
 7 | WARNING | [x] Unused use statement
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Comment by ekes** [comment]
This is no longer a thing with the updated entity browser modal; and autocomplete geo address which does update the map itself.

**Pull request closed without merging**